index.html

```diff
@@ -6,6 +6,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
-    
+    <h1>test</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?